bindgen/codegen/struct_layout.rs

```diff
@@ -156,9 +156,7 @@ impl<'a> StructLayoutTracker<'a> {
 
         self.latest_field_layout = Some(layout);
         self.last_field_was_bitfield = true;
-        // NB: We intentionally don't update the max_field_align here, since our
-        // bitfields code doesn't necessarily guarantee it, so we need to
-        // actually generate the dummy alignment.
```

Comment:
Hmm, yeah, I think 669dc1b62 fixed this effectively (this comment comes from 0d128822a, way before).

So looks great, thanks!